### ISSUE TITLE
ogc-server-statistics - tests with profile AllTests do not pass

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2076]
They require a `testdb` with the ogc-server-statistics schema loaded
```
georchestra/ogc-server-statistics (master *$%)$ ../mvn -PAllTests package
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building ogc-server-statistics 18.06-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- git-commit-id-plugin:2.1.10:revision (default) @ ogc-server-statistics ---
[INFO] 
[INFO] --- maven-resources-plugin:2.5:resources (default-resources) @ ogc-server-statistics ---
[debug] execute contextualize
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/fvanderbiest/workspace/georchestra/georchestra/ogc-server-statistics/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.3.2:compile (default-compile) @ ogc-server-statistics ---
[INFO] Compiling 1 source file to /home/fvanderbiest/workspace/georchestra/georchestra/ogc-server-statistics/target/classes
[INFO] 
[INFO] --- maven-resources-plugin:2.5:testResources (default-testResources) @ ogc-server-statistics ---
[debug] execute contextualize
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 8 resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.3.2:testCompile (default-testCompile) @ ogc-server-statistics ---
[INFO] Compiling 1 source file to /home/fvanderbiest/workspace/georchestra/georchestra/ogc-server-statistics/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:2.16:test (default-test) @ ogc-server-statistics ---
[INFO] Surefire report directory: /home/fvanderbiest/workspace/georchestra/georchestra/ogc-server-statistics/target/surefire-reports

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.georchestra.ogcservstatistics.log4j.DisableLoggingTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.117 sec <<< FAILURE! - in org.georchestra.ogcservstatistics.log4j.DisableLoggingTest
testOGCOperationLogging(org.georchestra.ogcservstatistics.log4j.DisableLoggingTest)  Time elapsed: 0.069 sec  <<< ERROR!
org.georchestra.ogcservstatistics.OGCServStatisticsException: org.georchestra.ogcservstatistics.dataservices.DataCommandException: ERROR: relation "ogcstatistics.ogc_services_log" does not exist
  Position : 49
	at org.georchestra.ogcservstatistics.dataservices.AbstractQueryCommand.execute(AbstractQueryCommand.java:95)
	at org.georchestra.ogcservstatistics.calculations.OGCServiceStatistics.execute(OGCServiceStatistics.java:359)
	at org.georchestra.ogcservstatistics.calculations.OGCServiceStatistics.list(OGCServiceStatistics.java:70)
	at org.georchestra.ogcservstatistics.log4j.DisableLoggingTest.testOGCOperationLogging(DisableLoggingTest.java:45)

Running org.georchestra.ogcservstatistics.log4j.OGCServicesAppenderTest
Tests run: 22, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.176 sec - in org.georchestra.ogcservstatistics.log4j.OGCServicesAppenderTest
Running org.georchestra.ogcservstatistics.log4j.OGCServiceParserTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec - in org.georchestra.ogcservstatistics.log4j.OGCServiceParserTest
Running org.georchestra.ogcservstatistics.calculations.OGCServiceStatisticsTest
java.lang.IllegalArgumentException: roles cannot be null
	at org.georchestra.ogcservstatistics.log4j.OGCServiceMessageFormatter.format(OGCServiceMessageFormatter.java:90)
	at org.georchestra.ogcservstatistics.log4j.OGCServiceMessageFormatter.format(OGCServiceMessageFormatter.java:61)
	at org.georchestra.ogcservstatistics.calculations.OGCServiceStatisticsTest.<clinit>(OGCServiceStatisticsTest.java:59)
```

**Comment by groldan** [comment]
@fvanderbiest @Vampouille @severo @pmauduit I've created PR #2392 which should fix all test errors against 18.06. Let me know if I should have done that agaist master instead so I can re issue it properly.

**Comment by groldan** [comment]
btw, I can't merge, so if it's ok some of you will have to. TIA.
